android/engine/src/main/java/org/smartregister/fhircore/engine/task/FhirCarePlanGenerator.kt

```diff
@@ -255,8 +282,17 @@ constructor(
             .filter { it.reference.startsWith(ResourceType.Task.name) }
             .mapNotNull { getTask(it.extractId()) }
             .forEach {
-              if (it.status.isIn(TaskStatus.REQUESTED, TaskStatus.READY, TaskStatus.INPROGRESS)) {
-                cancelTaskByTaskId(it.logicalId, "${carePlan.fhirType()} ${carePlan.status}")
+              if (
+                it.status.isIn(
+                  TaskStatus.REQUESTED,
+                  TaskStatus.READY,
+                  TaskStatus.INPROGRESS,
+                )
+              ) {
+                cancelTaskByTaskId(
+                  it.logicalId,
+                  "${carePlan.fhirType()} ${carePlan.status}",
```

Comment:
Use named parameters. These params are of the same type.

Reply:
change in code were not introduced by my commit and were there from before

Reply:
See my previous comment